backend/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.defaultFormatter": "denoland.vscode-deno"
```

Comment:
I vote for ignoring editor specific files to keep the repo clean

Reply:
Without editor specific files, VS Code seems to be not capable of distinguishing formatters for different folders.